hadoop-plugin/src/main/groovy/com/linkedin/gradle/azkaban/AzkabanDslYamlCompiler.groovy

```diff
@@ -97,20 +98,16 @@ class AzkabanDslYamlCompiler extends BaseCompiler {
   }
 
   /**
-   * Create and customize a Yaml object.
-   * DumperOptions.FlowStyle.BLOCK indents the yaml in the expected, most readable way.
+   * Instead of visiting workflows and namespaces in the same way the BaseNamedScopeContainer did.
```

Comment:
I don't quite follow the comment here. Can you elaborate?

Reply:
Looks like a typo - thanks!